python/cuda_parallel/cuda/parallel/experimental/typing.py

```diff
@@ -10,3 +12,9 @@ class DeviceArrayLike(Protocol):
     """
 
     __cuda_array_interface__: dict
+
+
+GpuStruct = Any
```

Comment:
(We had an offline discussion.)

If you decide to keep this code, could you please add a comment (terse is fine) to explain why we're currently using `Any`?

Reply:
Added a TODO